cmd/serve.go

```diff
@@ -16,8 +16,13 @@
 package cmd
 
 import (
+	"errors"
+	"github.com/vulcanize/gap-filler/pkg/mux"
```

Comment:
Really minor but can we move this external dep out of the builtins

Reply:
Done